gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
```

Comment:
I would recommend this version instead:
```
distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```

Gives IDE gradle sources for autocomplete

Reply:
Done